tests/integration/test_algorithms.py

```diff
@@ -19,7 +19,6 @@
 import strawberryfields as sf
 from strawberryfields.ops import *
 
-@pytest.mark.backends("fock","tf", "gaussian")
```

Comment:
I had put those marks in previously for the integration tests so that they skip the bosonic backend (since the backend wasn't linked to the frontend via an engine yet). Now that it is linked, many of these integration tests can be run with the bosonic backend.

Reply:
Got it! So just to make sure, are these tests still marked somehow? Just wondering since only marked test cases will be run by the CI.

Reply:
I'm not sure exactly how they're marked, but I know they are run when you submit a PR. I think the default in the conftest file is that the tests are run for all backends unless they are marked to only run on some.

Reply:
Right! They are indeed marked through the fixture of the engine.